### PR TITLE
reduce iteration count of btree test

### PR DESCRIPTION
**Pull request by bung87**
reduce iteration count.
doesn't seem need to be test very often.

### CODE REVIEW

File: tests/compiler/tbtrees.nim [hunk 1]
```diff
@@ -65,7 +65,7 @@ proc main =
         echo "got ", x, ", but expected ", iters - i
     echo b2.len
 
-  when true:
+  when false:
```
Comment (Araq) [inline tests/compiler/tbtrees.nim]:
```suggestion
  when true:
```

File: tests/compiler/tbtrees.nim [hunk 2]
```diff
@@ -65,7 +65,7 @@ proc main =
...
     var b2 = initBTree[int, string]()
     var t2 = initTable[int, string]()
     const iters = 100_000
```
Comment (Araq) [inline tests/compiler/tbtrees.nim]:
```suggestion
    const iters = 100
```

**Comment by Araq** [comment]
I messed up with github's UI, sorry. Please instead change `iters` to the value `100`.

**Comment by bung87** [comment]
reopened here https://github.com/nim-lang/Nim/pull/20491

**Pull request closed without merging**